windows/windows-impl/src/main/res/layout/activity_windows_waitlist.xml

```diff
@@ -91,6 +91,25 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/statusTitle" />
 
+            <FrameLayout
```

Comment:
This screen is used in 3 different states. Depending on the state, some elements are visible/hidden.
Notifyme component should only appear when user has joined the waitlist (1 of those 3 states), but it handles visibility internally. To avoid that, I'm wrapping the view with a framelayout, not ideal but works for this simple scenario (and we will remove the screen in the future).

Reply:
Have you tested this in all scenarios? The component uses Lifecycle to check for permissions, and it hooks into it `onAttach`. I'm worried that you might be missing hooks if the FL is one by default.

Reply:
Moving answer to Asana. I tested it and works fine, but I want to double-check check I'm not missing a scenario I don't know.